forms-flow-web/src/components/FOI/customComponents/Records/index.js

```diff
@@ -1550,437 +2078,532 @@ export const RecordsLog = ({
+    };
+    const [popoverOpen, setPopoverOpen] = useState(false);
+    const [anchorPosition, setAnchorPosition] = useState(null);
 
-    return (
-      <MenuItem
-        onClick={() => {
-          handleDelete();
-          setPopoverOpen(false);
-         }}
-      >
-        Delete
-      </MenuItem>
-    )
+    const ReplaceMenu = () => {
+      return (
+        <MenuItem
+          onClick={() => {
+            handleReplace();
+            setPopoverOpen(false);
+          }}
+        >
+          Replace
+        </MenuItem>
+      );
+    };
 
-  }
+    const DeleteMenu = () => {
+      return (
+        <MenuItem
+          onClick={() => {
+            handleDelete();
+            setPopoverOpen(false);
+          }}
+        >
+          Delete
+        </MenuItem>
+      );
+    };
 
-  // const AddMenuItems = () => {
-  //   if (showReplace(record.category))
-  //     return (<ReplaceMenu />)
-  //   return (<DeleteMenu />)
-  // }
+    // const AddMenuItems = () => {
+    //   if (showReplace(record.category))
+    //     return (<ReplaceMenu />)
+    //   return (<DeleteMenu />)
+    // }
 
-  const ActionsPopover = ({RestrictViewInBrowser, record}) => {
-    const conversionFormats = useSelector((state) => state.foiRequests.conversionFormats)
-    return (
-      <Popover
-        anchorReference="anchorPosition"
-        anchorPosition={
-          anchorPosition && {
-            top: anchorPosition.top,
-            left: anchorPosition.left,
-          }
-        }
-        open={popoverOpen}
-        anchorOrigin={{
-          vertical: "top",
-          horizontal: "left",
-        }}
-        transformOrigin={{
-          vertical: "top",
-          horizontal: "left",
-        }}
-        onClose={() => setPopoverOpen(false)}
-      >
-        <MenuList>
-{!RestrictViewInBrowser ?
-        <MenuItem
-            onClick={() => {
-                handleView();
-                setPopoverOpen(false);
-            }}
-          >
-            View
-          </MenuItem>
-          :""}
-          { (!record.attributes?.isattachment || record.attributes?.isattachment  === undefined) && <MenuItem
-            onClick={() => {
-                handleReplace();
-                setPopoverOpen(false);
-            }}
-          >
-            Replace Manually
-          </MenuItem>}
-          { record.attributes?.isattachment && <MenuItem
-            onClick={() => {
-                 handleReplaceAttachment()
-                setPopoverOpen(false);
-            }}
-          >
-            Replace Attachment
-          </MenuItem>}
-          {record.originalfile!=''  && record.originalfile!=undefined  && <MenuItem
-            onClick={() => {
-                handleDownloadoriginal();
-                setPopoverOpen(false);
-            }}
-          >
-            Download Original
-          </MenuItem>
+    const ActionsPopover = ({ RestrictViewInBrowser, record }) => {
+      return (
```

Comment:
const conversionFormats = useSelector((state) => state.foiRequests.conversionFormats)
removed above code as it is not needed any more